Error_during_API_call_API_call_failed_{_error_{@20241219_113507.md

```
@@ -0,0 +1,1148 @@
+Making API call
+API call failed
+Error during API call: API call failed: {
+  "error": {
+    "message": "This model's maximum context length is 128000 tokens. However, you requested 128007 tokens (2133 in the messages, 125874 in the completion). Please reduce the length of the messages or completion.",
+    "type": "invalid_request_error",
+    "param": "messages",
+    "code": "context_length_exceeded"
+  }
+}
+Traceback (most recent call last):
+  File "/home/azureuser/simple/core/ai_service.py", line 338, in _make_api_call_with_retry
```

Markdownlint (0.37.0)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)